snmp/tests/test_profiles.py

```diff
@@ -412,6 +412,8 @@ def test_f5(aggregator):
             )
 
     aggregator.assert_metric('snmp.sysUpTimeInstance', count=1)
+    aggregator.assert_metric('snmp.cpu.usage', count=2)
+
```

Comment:
```suggestion
```

It seems this metric is already asserted above in this test.

Is there a reason for having it?

Reply:
Asserted here https://github.com/DataDog/integrations-core/blob/f75a95a619b301b3e694a401264bee8028f70bf5/snmp/tests/test_profiles.py#L292

Reply:
I must have had failing tests because of it, but for now the problem is not present, so I deleted this assertion